src/modules/auth/ui/AuthFormModal/AuthFormModal.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC, ReactNode } from 'react';
+
+import { Popup } from '@/ui/Popup/Popup';
+
+type AuthFormModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  children: ReactNode;
+};
+
+const AuthFormModal: FC<AuthFormModalProps> = ({
+  isOpen,
+  onClose,
+  children,
+}) => {
+  return (
+    <Popup isOpen={isOpen} onClose={onClose}>
+      {children}
+    </Popup>
+  );
+};
+
+export default AuthFormModal;
```

Comment:
```suggestion
export const AuthFormModal: FC<AuthFormModalProps> = ({
  isOpen,
  onClose,
  children,
}) => {
  return (
    <Popup isOpen={isOpen} onClose={onClose}>
      {children}
    </Popup>
  );
};
```

Честно, не понимаю, почему вас всех на дефолтные экспорты потянуло